add list part

maxNia opened the pull request:
code working fine!
video in telegram

StrNikita committed:
add interaction with front

add bodyParser
avoid CORS
add getAll route/controller/service

StrNikita committed:
update config files, update contact files, add list files and update list.model file, add third tableModel

StrNikita committed:
small refactoring

StrNikita committed:
delete user module

StrNikita committed:
just change main folder name(never mind)

StrNikita committed:
move some files to renamed folder

CatDidenko reviewed (changes requested):
Please, divide relations to the separate files. Contacts relations move to contact file, List relations move to the list model.

CatDidenko commented on src/listOfContacts/listOfContacts.model.ts:
```diff
@@ -0,0 +1,17 @@
+import { DataTypes, Model } from 'sequelize';
+import sequelize from "../config/database";
```
single quotes

maxNia replied:
updated

StrNikita committed:
moved relations to server.ts

CatDidenko commented on contactService/config/server.ts:
```diff
@@ -0,0 +1,35 @@
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+
```
Remove extra empty lines

maxNia replied:
updated

CatDidenko commented on contactService/contact/contact.controller.ts:
```diff
@@ -0,0 +1,20 @@
+  try {
+    const contacts = await contactService.getContacts();
+    res.status(200).send(contacts)
+  } catch(e: any) {
```
remove any

maxNia replied:
removed

CatDidenko commented on contactService/email/mail.ts:
```diff
@@ -0,0 +1,50 @@
+routers.get('/mail', mailToAll);
+
+async function mailToAll(req: any, res: any, next: any) {
+  console.log('hello');
```
remove console.log

CatDidenko replied:
Actually, this functionality should be in the separate microservice (separate express project)

CatDidenko replied:
Please, remove this file

maxNia replied:
deleted

CatDidenko commented on contactService/email/mail.ts:
```diff
@@ -0,0 +1,50 @@
+import { Router } from "express";
```
single quotes, please

maxNia replied:
file deleted

StrNikita committed:
small refactoring

everywhere single quotes
remove any from catch sections
remove empty lines
remove email file
